.github/workflows/build.yml

```diff
@@ -108,11 +108,13 @@ jobs:
           github.event_name == 'push'
         run: |
           gh auth status
-          gh release delete v0.0.3 -y || true
+          VERSION="$(cat setup.py | grep 'version=' | cut -d "'" -f 2)"
```

Comment:
If we are okay with "running" setup.py we can do `python setup.py --version`

Reply:
I applied your suggestion.